src/sqlacodegen/generators.py

```diff
@@ -1224,7 +1235,10 @@ def get_type_qualifiers() -> tuple[str, TypeEngine[Any], str]:
             return "".join(pre), column_type, "]" * post_size
 
         def render_python_type(column_type: TypeEngine[Any]) -> str:
-            python_type = column_type.python_type
+            if isinstance(column_type, DOMAIN):
+                python_type = column_type.data_type.python_type
+            else:
+                python_type = column_type.python_type
```

Comment:
Add a blank line after a control block.
```suggestion
                python_type = column_type.python_type

```